components/BottomSheet/src/private/MDCSheetContainerView.m

```diff
@@ -312,27 +304,36 @@ - (BOOL)draggableView:(__unused MDCDraggableView *)view
   }
 }
 
+- (BOOL)canExtend {
+  return self.preferredSheetHeight != [self maximumSheetHeight];
+}
+
 - (void)draggableView:(__unused MDCDraggableView *)view
     draggingEndedWithVelocity:(CGPoint)velocity {
   MDCSheetState targetState;
-  if (self.preferredSheetHeight == [self maximumSheetHeight]) {
-    // Cannot be extended, only closed.
-    targetState = (velocity.y >= 0 ? MDCSheetStateClosed : MDCSheetStatePreferred);
-  } else {
+  if ([self canExtend]) {
     CGFloat currentSheetHeight = CGRectGetMaxY(self.bounds) - CGRectGetMinY(self.sheet.frame);
     if (currentSheetHeight >= self.preferredSheetHeight) {
-      targetState = (velocity.y >= 0 ? MDCSheetStatePreferred : MDCSheetStateExtended);
+      if (velocity.y >= 2000) { // Hard toss to close.
```

Comment:
Is `2000` a threshold that UIKit documents somewhere? Is it just based on multiple tests? Is it invariant across devices (iPhone 5 vs. iPad Pro)?

Reply:
This is a tough one and one that I'd love for us to get some UX insight on.

Reply:
I've removed this additional behavior for now.